mmv1/third_party/terraform/services/apigee/resource_apigee_environment_update_test.go

```diff
@@ -0,0 +1,115 @@
+package apigee_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccApigeeEnvironment_apigeeEnvironmentUpdateTest(t *testing.T) {
```

Comment:
What exactly is this test updating? Can you make this a 2 steps test to show the update?

Reply:
Do you mean splitting `resource.TestStep` within one test case, or separating it into a different `func`?
Just wanted to confirm, thanks!

Reply:
Looks like I missed `testAccApigeeEnvironment_apigeeEnvironmentBasicTestExample` in the test first step. Is there any particular reason to skip the vcr test here?

Reply:
To be honest, I don’t fully understand the exact reason for skipping VCR test.
In the past [PR](https://github.com/GoogleCloudPlatform/magic-modules/pull/11479), we had a same situation.

I’ll remove the VCR skip for now and see if it works.